src/components/PeopleTable/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { PersonLink } from '../../PersonLink';
+import type { Person } from '../../../types';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
```

Comment:
If you need to get a lot of fields from an object, it is a little more convenient to destructure such an object